scss/main.scss

```diff
@@ -301,11 +310,148 @@ body {
 }
 
 // *** what can you do section ***
+
+@keyframes expand {
+  0% {
+      height: auto;
+  }
+  100% {
+      height: auto;
+  }
+}
+@keyframes unexpand {
+  0% {
+      height: auto;
+  }
+  100% {
+      height: auto;
+  }
+}
+
 a:hover {
   text-decoration: none;
 }
 
 .what-can-you-do {
+/* text box */
+
+  .content-heading {
+    font-size: 2.5rem;
+    font-weight: bold;
+    margin-bottom: 1rem;
+  }
+
+  .arrow-img {
+    margin-left: 0.5rem;
+  }
+
+  .expand-btn {
```

Comment:
can you style this as a button? use our button mixin so it matches the other buttons on the site:
<img width="331" alt="Screen Shot 2021-08-15 at 11 35 05 AM" src="https://user-images.githubusercontent.com/62481939/129483913-c72ca076-5c7d-4b88-9db5-381d38cb9d93.png">